docs/source/ubuntu-opens-install.rst

```diff
@@ -387,6 +389,12 @@ perform the following steps:
    generate the configuration for a single peer connection, which you can
    duplicate and update for each compute host in your mesh.
 
+   To maintain connectivity between VMs if BIRD crashes or is reloaded, enable
+   BIRD graceful restart by adding `-R` to `BIRD_ARGS` in `/etc/bird/envvars`
+   (you may need to uncomment this option).  Please not that this will not work
+   if BIRD is stopped normally - if you need that function, you should modify
+   the upstart job to kill BIRD on stop.
+
```

Comment:
Same comment as for Red Hat - this is restart and we need to provide the upstart instructions.